src/components/custombutton-large.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+interface CustomButtonProps {
```

Comment:
We can extend the interface here with `HTMLAttributes<HTMLButtonElement>`, that means people can pass in all of the standard props like `onBlur` or `disabled`.

LMK if you need help doing this, it will require some changes to the function declaration too.